datasets/socialgene.yaml

```diff
@@ -0,0 +1,42 @@
+Name: SocialGene RefSeq Databases
+Description: Precomputed SocialGene Neo4j graph databases of various sizes built from RefSeq genomes and MIBiG BGCs. 
+Documentation: https://socialgene.github.io/precomputed_databases/2023_v0.4.1/aws
```

Comment:
This page is 404'ing for me @chasemc

Reply:
Thanks, I've fixed the url

Reply:
Thank you. Please update the branch and I will merge this in. It seems to be out of date with main.